<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportSettingsDialog</class>
 <widget class="QDialog" name="ExportSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Video</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="cameraTab">
      <attribute name="title">
       <string>Single Camera</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="exportMediaContentsWidget" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>16</number>
          </property>
          <property name="spacing">
           <number>16</number>
          </property>
          <item row="0" column="1">
           <widget class="QStackedWidget" name="mediaExportSettingsWidget">
            <property name="minimumSize">
             <size>
              <width>280</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>280</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="nx::vms::client::desktop::ExportMediaSettingsWidget" name="exportMediaSettingsPage"/>
            <widget class="nx::vms::client::desktop::BookmarkOverlaySettingsWidget" name="bookmarkSettingsPage"/>
            <widget class="nx::vms::client::desktop::TimestampOverlaySettingsWidget" name="timestampSettingsPage"/>
            <widget class="nx::vms::client::desktop::ImageOverlaySettingsWidget" name="imageSettingsPage"/>
            <widget class="nx::vms::client::desktop::TextOverlaySettingsWidget" name="textSettingsPage"/>
            <widget class="nx::vms::client::desktop::InfoOverlaySettingsWidget" name="infoSettingsPage"/>
            <widget class="nx::vms::client::desktop::RapidReviewSettingsWidget" name="rapidReviewSettingsPage"/>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QWidget" name="buttonsWidget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="nx::vms::client::desktop::SelectableTextButton" name="cameraExportSettingsButton">
               <property name="text">
                <string>Export Settings</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="transcodingButtonsWidget" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="spacing">
                 <number>4</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="nx::vms::client::desktop::SelectableTextButton" name="bookmarkButton">
                  <property name="text">
                   <string notr="true">&lt;Bookmark Info&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="nx::vms::client::desktop::SelectableTextButton" name="timestampButton">
                  <property name="text">
                   <string notr="true">&lt;Timestamp&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="nx::vms::client::desktop::SelectableTextButton" name="imageButton">
                  <property name="text">
                   <string notr="true">&lt;Image&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="nx::vms::client::desktop::SelectableTextButton" name="textButton">
                  <property name="text">
                   <string notr="true">&lt;Text&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="nx::vms::client::desktop::SelectableTextButton" name="infoButton">
                  <property name="text">
                   <string notr="true">&lt;Info&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="nx::vms::client::desktop::SelectableTextButton" name="speedButton">
                  <property name="text">
                   <string notr="true">&lt;Speed&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>304</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="overlayButtonsWidget" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <property name="spacing">
                 <number>4</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QFrame" name="mediaFrame">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="1" column="1">
              <widget class="nx::vms::client::desktop::AsyncImageWidget" name="mediaPreviewWidget" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="1">
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="2">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="nx::vms::client::desktop::FilenamePanel" name="mediaFilenamePanel" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="mediaAlertsWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="weakMediaAlertsLayout">
            <property name="spacing">
             <number>1</number>
            </property>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="severeMediaAlertsLayout">
            <property name="spacing">
             <number>1</number>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutTab">
      <attribute name="title">
       <string>Multi Video</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="exportLayoutContentsWidget" native="true">
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>16</number>
          </property>
          <property name="topMargin">
           <number>16</number>
          </property>
          <property name="rightMargin">
           <number>16</number>
          </property>
          <property name="bottomMargin">
           <number>16</number>
          </property>
          <property name="spacing">
           <number>16</number>
          </property>
          <item row="0" column="0">
           <widget class="QFrame" name="layoutFrame">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="1">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>228</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="1">
              <widget class="nx::vms::client::desktop::AsyncImageWidget" name="layoutPreviewWidget" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>228</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="1">
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QStackedWidget" name="layoutExportSettingsWidget">
            <property name="minimumSize">
             <size>
              <width>280</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>280</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="nx::vms::client::desktop::ExportLayoutSettingsWidget" name="exportLayoutSettingsPage"/>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QWidget" name="layoutButtonsWidget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>4</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="nx::vms::client::desktop::SelectableTextButton" name="layoutExportSettingsButton">
               <property name="text">
                <string>Export Settings</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="nx::vms::client::desktop::FilenamePanel" name="layoutFilenamePanel" native="true"/>
       </item>
       <item>
        <widget class="QWidget" name="layoutAlertsWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="weakLayoutAlertsLayout">
            <property name="spacing">
             <number>1</number>
            </property>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="severeLayoutAlertsLayout">
            <property name="spacing">
             <number>1</number>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::ExportMediaSettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/export/widgets/export_media_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::ExportLayoutSettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/export/widgets/export_layout_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::SelectableTextButton</class>
   <extends>QPushButton</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/selectable_text_button.h</header>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::TimestampOverlaySettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/export/widgets/timestamp_overlay_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::ImageOverlaySettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/export/widgets/image_overlay_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::TextOverlaySettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/export/widgets/text_overlay_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::RapidReviewSettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/export/widgets/rapid_review_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::FilenamePanel</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/filename_panel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::AsyncImageWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/async_image_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::BookmarkOverlaySettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/export/widgets/bookmark_overlay_settings_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::InfoOverlaySettingsWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/export/widgets/info_overlay_settings_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
